python-avd/pyavd/_anta/logs.py

```diff
@@ -91,7 +91,11 @@ class LogMessage(Enum):
 
     # WAN-specific messages
     PATH_GROUP_NO_STUN_INTERFACE = "path group {path_group} skipped - No STUN client interfaces found"
+    PATH_GROUP_NO_LOCAL_INTERFACES = "path group {path_group} skipped - No local interfaces found"
     PATH_GROUP_NO_STATIC_PEERS = "path group {path_group} skipped - No static peers configured"
+    PATH_GROUP_STATIC_PEER_NO_ROUTER_IP = "static peer {static_peer} skipped - No router ip configured"
+    PATH_GROUP_STATIC_PEER_NO_IPV4_ADDRESSES = "static peer {static_peer} skipped - No ipv4 addresses configured"
```

Comment:
Would it be useful to add the path group to these?
```suggestion
    PATH_GROUP_STATIC_PEER_NO_ROUTER_IP = "path group {path_group} static peer {static_peer} skipped - No router ip configured"
    PATH_GROUP_STATIC_PEER_NO_IPV4_ADDRESSES = "path group {path_group} static peer {static_peer} skipped - No ipv4 addresses configured"
```

Reply:
updated